frontend/src/metabase/visualizations/echarts/cartesian/option/index.ts

```diff
@@ -10,43 +10,53 @@ import { getAxesFormatters } from "metabase/visualizations/echarts/cartesian/opt
 
 import { getChartGrid } from "./grid";
 import { getGoalLineSeriesOption } from "./goal-line";
+import { getTrendLineOptionsAndDatasets } from "./trend-line";
```

Comment:
Wdyt about moving the trend line dataset generation to the chart model so that it will be memoized?